bin/check-aggregate.rb

```diff
@@ -129,6 +129,13 @@ class CheckAggregate < Sensu::Plugin::Check::CLI
          description: 'Ignore severities, all non-ok will count for critical, critical_count, warning and warning_count option',
          boolean: true,
          default: false
+ 
+  option :switch_output,
```

Comment:
match indentation

Reply:
Also would this better be served with something more generic like debug?

Reply:
I agree, `--debug` or `--verbose` seems to make more sense here.